### ISSUE TITLE
BasicModel from lane command line

### ISSUE THREAD

**Issue by flathat** [CORE-POS/IS4C#611]
What should I be looking for when, lane-side:
```
IS4C/pos/is4c-nf/lib/models$ php ./BasicModel.php --update opdata ./op/DepartmentsModel.php              
Error: class 'DepartmentsModel' does not exist
```
when op/DepartmentsModel.php
```
namespace COREPOS\pos\lib\models\op;
use COREPOS\pos\lib\models\BasicModel;
/**
  @class DepartmentsModel
*/
class DepartmentsModel extends BasicModel
{
```

while similar OK server-side:
```
IS4C/fannie/classlib2.0/data/models$ php ./BasicModel.php --update core_op ./op/DepartmentsModel.php
==========================================
Checking table core_op.departments
==========================================
==========================================
Check complete. Need to adjust 0 columns  0 indexes.
==========================================
```
when op/DepartmentsModel.php
```
/**
  @class DepartmentsModel
*/
class DepartmentsModel extends BasicModel
{
```

**Comment by flathat** [comment]
CORE 1.8
Install > DBUpdates
>To apply changes Click Here or run the following command:
php /var/www/IS4C/pos/is4c-nf/lib/models/BasicModel.php --update opdata COREPOS\pos\lib\models\op\ProductsModel

```
$ pwd
/var/www/IS4C/pos/is4c-nf/lib/models
$ ls op/ProductsModel.php
op/ProductsModel.php

namespace COREPOS\pos\lib\models\op;
use COREPOS\pos\lib\models\BasicModel;
class ProductsModel extends BasicModel
{

$ php /var/www/IS4C/pos/is4c-nf/lib/models/BasicModel.php --update opdata COREPOS\pos\lib\models\op\ProductsModel
Error: file 'COREPOSposlibmodelsopProductsModel.php' does not exist
# quoted
$ php /var/www/IS4C/pos/is4c-nf/lib/models/BasicModel.php --update opdata 'COREPOS\pos\lib\models\op\ProductsModel'
Error: file 'COREPOS\pos\lib\models\op\ProductsModel.php' does not exist
```

**Comment by gohanman** [comment]
RE: First post:

I have no idea. I can't replicate that at all. This is what I get in version-1.8, version-1.9, and master:
```$ php ./BasicModel.php --update lane_test ./op/DepartmentsModel.php
==========================================
Checking table lane_test.departments
==========================================
==========================================
Check complete. Need to add 0 columns  0 indexes.
==========================================
```

RE: CLI instructions: there is an issue to be addressed there in version-1.9 and master, but in version-1.8 those model classes do not have namespaces (e.g., https://github.com/CORE-POS/IS4C/blob/version-1.8/pos/is4c-nf/lib/models/op/ProductsModel.php)

**Comment by flathat** [comment]
Maybe it will come around when I install 1.9.

**Comment by gohanman** [comment]
I'm not sure that's the case. None of this exists before 1.9, so if your lane models have namespaces then you've already merged 1.9 or some subset of its commits into your code.
